server/channels/api4/import_test.go

```diff
@@ -127,7 +127,10 @@ func TestImportInLocalMode(t *testing.T) {
 
 	received, _, err := th.SystemAdminClient.CreateJob(context.Background(), job)
 	require.NoError(t, err)
-	defer th.App.Srv().Store().Job().Delete(received.Id)
+	defer func() {
+		_, err := th.App.Srv().Store().Job().Delete(received.Id)
```

Comment:
The new `err` variable shadows an existing one. Please just reuse it.
```suggestion
		_, err = th.App.Srv().Store().Job().Delete(received.Id)
```